src/flownet/data/from_flow.py

```diff
@@ -153,7 +156,7 @@ def _production_data(self) -> pd.DataFrame:
             * Improve robustness pf setting of Phase and Type.
 
         """
-        keys = ["WOPR", "WGPR", "WWPR", "WBHP", "WTHP", "WGIR", "WWIR", "WSTAT"]
+        keys = ["WOPR", "WGPR", "WWPR", "WOPT", "WGPT", "WWPT", "WBHP", "WTHP", "WGIR", "WWIR", "WGIT", "WWIT", "WSTAT"]
```

Comment:
You're adding `WGIT` and `WWIT` here but not in the documentation a few lines earlier and neither are they added in `src/flownet/config_parser/_config_parser.py`. Is that intentional?

Should you read those values at all? You can reconstruct the cumulative at any given date by calculating the cumulative sum over the rates (this would also be easier to use later on in other data sources)? Or do we expect big differences based on differences in reporting steps?

Reply:
The rates are instantaneous, and depending on the report step length we can get a quite different result reading the actual cumulative volumes and calculating them from rates. Over the years when doing uncertainty studies I have always calculated average yearly or monthly rates from cumulative volumes - for history matching purposes I guess the instantaneous rates are important - what I am saying is that we may need to import both then.

Reply:
We may indeed not be using WGIT and WWIT, unless we prescribe BHP instaed of the rates during the history matching. I thought it would be good for completeness to just add them. I forgot about adding them to **src/flownet/config_parser/_config_parser.py.**